GLMakie/src/GLAbstraction/GLTypes.jl

```diff
@@ -289,11 +297,14 @@ mutable struct RenderObject{Pre}
     prerenderfunction::Pre
     postrenderfunction
     id::UInt32
+    requires_update::Bool
```

Comment:
I think we should consistently use the same type for `requires_update`, and I don't think we need an observable, so maybe use `Base.RefValue` consistently instead?

Reply:
Ah, I  see, some do need to be an observable..So maybe use Observable consitently?

Reply:
For `GPUArray`we either need an observable to forward to the information to the renderobject or we need to search through a bunch of stuff every potential frame. Or we need to make their updates aware of the renderobject but that might be a lot of work?

Otherwise I went with Bool since the structs are already mutable. I think that's a bit faster than a Refvalue in a mutable struct too...